### PR TITLE
RHBPMS-4804: The new process editor cannot make straight lines

### PR DESCRIPTION
**Pull request by Josephblt**
Fix for the ticket https://issues.jboss.org/browse/RHBPMS-4804.
@romartin @manstis @hasys

### CODE REVIEW

**Comment by romartin** [comment]
ok to test

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-shapes/kie-wb-common-stunner-shapes-client/src/main/java/org/kie/workbench/common/stunner/shapes/client/view/ConnectorView.java [hunk 1]
```diff
@@ -41,14 +41,14 @@ private ConnectorView(final Object[] line) {
               (MultiPathDecorator) line[2]);
     }
 
-    private static Object[] createLine(final double... points) {
+    static Object[] createLine(final double... points) {
         // The head decorator must be not visible, as connectors are unidirectional.
         final MultiPath head = new MultiPath()
-                .M(1,
-                   2)
+                .M(0,
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-shapes/kie-wb-common-stunner-shapes-client/src/main/java/org/kie/workbench/common/stunner/shapes/client/view/ConnectorView.java]: Why do we need the extra ```M``` and ```L```? Would ```M(0, 0).Z()``` not be enough?

Comment (Josephblt) [reply kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-shapes/kie-wb-common-stunner-shapes-client/src/main/java/org/kie/workbench/common/stunner/shapes/client/view/ConnectorView.java]: I think it wil, to be honest I think an empty multipath will do the trick as well. Will test it as soon as possible.

**Comment by hasys** [comment]
Jenkins execute full downstream build

**Comment by Josephblt** [comment]
Jenkins execute full downstream build

**Comment by Josephblt** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
@hasys please need your review here, as this still needs to be cherry-picked to 7.3.x before end of week... thanks!

**Comment by hasys** [comment]
Jenkins execute full downstream build

**Pull request merged**